### DISCUSSION TITLE
How is the characters.json made?

### DISCUSSION THREAD

**Discussion by ginqus** [IdreesInc/Monocraft#134] [Q&A] [answered]
I wanted to try to make a Minecraft font that is not monospaced. I know there are already, but none of them have as many glyphs as Monocraft does. So, how is the characters.json made? Like, there's no way someone would manually write ones and zeros for pixels, right? Is there a way to automate or simplify this process? I didn't find anything for this inside the /src folder, so, if you have the time, could you please share how you achieved this? Thanks! :)

**Comment by Ciubix8513** [comment, marked as answer]
Nope, all json files are created manually, except for diacritics, which are added automatically

**Reply by ginqus** [reply]
Oh, that's a huge dedication then. Well, thanks anyway!
